app/presenters/publishing_api/payload_builder/brexit_content_notices.rb

```diff
@@ -14,49 +14,46 @@ def initialize(item)
       def call
         return {} if no_brexit_content_notices?
 
-        show_no_deal_notice? ? brexit_no_deal_notice_payload : brexit_current_state_notice_payload
+        show_no_deal_notice? ? no_deal_notice_payload : current_state_notice_payload
       end
 
     private
 
-      def brexit_no_deal_notice_payload
-        { brexit_no_deal_notice: links }
+      def no_deal_notice_payload
+        { brexit_no_deal_notice: no_deal_links }
       end
 
-      def brexit_current_state_notice_payload
+      def current_state_notice_payload
         { brexit_current_state_notice: current_state_links }
       end
 
       def no_brexit_content_notices?
         hide_no_deal_notice? && hide_current_state_notice?
       end
 
-      def links
-        non_blank_links.map do |link|
-          {
-            title: link.title,
-            href: href(link),
-          }
-        end
+      def no_deal_links
+        links_for_payload(item.brexit_no_deal_content_notice_links)
       end
 
       def current_state_links
-        non_blank_current_state_links.map do |link|
-          {
-            title: link.title,
-            href: href(link),
-          }
-        end
+        links_for_payload(item.brexit_current_state_content_notice_links)
       end
 
-      def non_blank_links
-        item.brexit_no_deal_content_notice_links.reject do |link|
-          link.title.blank? && link.url.blank?
+      def links_for_payload(links)
+        non_blank_links(links).map do |link|
+          to_hash(link)
         end
       end
 
-      def non_blank_current_state_links
-        item.brexit_current_state_content_notice_links.reject do |link|
+      def to_hash(link)
+        {
+          title: link.title,
+          href: href(link),
+        }
+      end
+
+      def non_blank_links(links)
+        links.reject do |link|
           link.title.blank? && link.url.blank?
```

Comment:
Wondering if this should be an `||` ? If either the title or url are blank it's probably bad?